components/Footer/Footer.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Image from "next/image";
+import { FooterPrivacy } from "./FooterPrivacy";
+import styles from "@styles/footer.module.scss";
+
+function Footer() {
+  return (
+    <footer className={`container ${styles.footer}`} id="footer" name="footer">
+      <div className="row justify-content-between mb-3">
```

Comment:
w sumie to ma byc praiwe 50px, wiem ze pierdola ale możemy to robić tak jak jest żeby wszystko było spójne